api/v1alpha2/inferencepool_types.go

```diff
@@ -95,9 +95,14 @@ type Extension struct {
 
 // ExtensionReference is a reference to the extension.
 //
+// Connections to this extension MUST use TLS by default. Implementations MAY
+// provide a way to customize this connection to use a different protocol or
+// custom TLS configuration.
```

Comment:
```suggestion
// provide a way to customize this connection to use plain-text, a different protocol or
// custom TLS configuration.
```

Reply:
Or "cleartext" (which is the opposite of "ciphertext" whereas "plain-text" might be interpreted as the opposite of "binary-encoded").